### DISCUSSION TITLE
xRechnung Version 2.1.1

### DISCUSSION THREAD

**Discussion by J-23** [stephanstapel/ZUGFeRD-csharp#174] [General]
Hello Stephan,
My question is about the x Rechnung Version 2.1.1 . ( at 01.02.2022)
When will you plan to update the new version for x Rechnung?
Thanks,
Ivan

**Comment by stephanstapel** [comment]
Hi Ivan,
current version is 2.2.0: https://www.xoev.de/xrechnung-16828
As far as I understand the changes since 2.0, there are mostly business rules that have been updated. Is there any impact you are experiencing?

**Comment by stephanstapel** [comment]
According to @TomcatBB's issue: https://github.com/stephanstapel/ZUGFeRD-csharp/issues/175 I just committed the necessary change.
